utils/Constants.sol

```diff
@@ -283,16 +284,10 @@ contract Constants {
         vm.label(value, key);
     }
 
-    function addCauldron(
-        string memory chain,
-        string memory name,
-        address value,
-        uint8 version,
-        bool deprecated
-    ) public {
+    function addCauldron(string memory chain, string memory name, address value, uint8 version, bool deprecated) public {
```

Comment:
Did you change the .prettierrc stuff recently? This line seems to only be 122 chars long. Curious why it wasn't formatted like this previously.

Reply:
didn't changed any formatting settings for ages but I don't have auto format on save because I don't want to format legacy code like bento and cauldrons, tho we could configure it so it ignores certain files.
I just never took the time to actually do it